assets/js/theme/common/carousel/utils/activatePlayPauseButton.js

```diff
@@ -0,0 +1,50 @@
+import { throttle } from 'lodash';
+
+const PLAY_ACTION = 'slickPlay';
+const PAUSE_ACTION = 'slickPause';
+const ACTIVATED_CLASS = 'js-play-pause-activated';
+
+export default (carousel, slidesQuantity, context) => {
```

Comment:
If we are using context widely, could we extend it with all of relevant data?

Reply:
As we discussed when util needs only one context value destructuring was replaced with relevant data